spec/paperclip/attachment_spec.rb

```diff
@@ -36,7 +36,7 @@
     rebuild_class styles: {
       small: '100x>',
       large: '500x>',
-      original: '42x42#' 
+      original: '42x42#'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.